app/src/renderer/vuex/modules/delegation.js

```diff
@@ -135,54 +138,88 @@ export default ({ node }) => {
       let candidates = await dispatch(`getDelegates`)
       return dispatch(`getBondedDelegates`, candidates)
     },
+    async getMyDelegates({ rootState, commit }) {
```

Comment:
this code is not necessary. we already have all validators of a delegator in commitedDelegations. this just creates an additional request.

Reply:
commented

Reply:
Thank you